src/main/java/net/minecraftforge/event/entity/player/PlayerEvent.java

```diff
@@ -462,6 +463,50 @@ public ItemStack getSmelting()
         }
     }
 
+    /**
+     *
+     * Event fired before the player is accepted to the world
+     *
+     */
+    @Cancelable
+    public static class PrePlayerLoginEvent extends PlayerEvent {
```

Comment:
Wrong formatting here still.
Also please document what cancelling does.

Reply:
Added javadoc comments at the top of the class and on the setCancelled override (and fixed formatting)